### PR TITLE
Frontend routes scaffold and basic header

### PR DESCRIPTION
**Pull request by jjessieshang**
### Description
---
Established route structure react-router-dom. Created routes to essential volunteer pages (classes, profile, and hours). Also created a basic navigation bar with the BC brain logo.

### Testing
---
Tested locally. All links functional.

### CODE REVIEW

**Review by leowang801 — APPROVED** [review]
Everything looks good

File: frontend/public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Labby</title>
+    <title>NERUON</title>
```
Comment (JoshFung) [inline frontend/public/index.html:27]:
```suggestion
    <title>NEURON</title>
```

Comment (JoshFung) [reply frontend/public/index.html:27]: LGTM besides this tiny typo 👍

**Review by jansm04 — APPROVED** [review]
LGTM!

**Pull request merged**